doc/Language/haskell-to-p6.pod6

```diff
@@ -537,6 +537,21 @@ compare it to let/in and where constructs maybe?
 
 TODO
 
+=head1 Tail Call Optimization
+
+=head2 Tail Call Optimization or Tail Call Elimination
+
```

Comment:
Hum, why create a level-1 section with nothing buth a level-2 section? Wouldn't it be more convenient to just have one of them?